src/CoreWCF.Http/src/CoreWCF/WSFederationHttpBinding.cs

```diff
@@ -0,0 +1,219 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.ComponentModel;
+using CoreWCF.Channels;
+
+namespace CoreWCF
+{
+
+    public class WSFederationHttpBinding : WSHttpBindingBase
+    {
+        static readonly MessageSecurityVersion WSMessageSecurityVersion = MessageSecurityVersion.WSSecurity11WSTrustFebruary2005WSSecureConversationFebruary2005WSSecurityPolicy11BasicSecurityProfile10;
+
```

Comment:
Missing accessibility modifies. Also naming violations on fields, and remove `this.` from member usage.